core/controllers/suggestion.py

```diff
@@ -148,6 +148,24 @@ def post(self):
 
         self.render_json(self.values)
 
+    @acl_decorators.can_update_suggestions
+    def put(self, suggestion_id):
```

Comment:
Can you add a TODO comment here to note that this currently only handles translation suggestions, but it will be updated to handle suggestions more generally?

Reply:
Done